documentation/cobigen-intellij-installation.asciidoc

```diff
@@ -0,0 +1,62 @@
+:toc:
+toc::[]
+
+
+= Installation
+
+
+== IntelliJ setup
+
+If you want to use IntelliJ for CobiGen project, you can follow the steps below:
+
+. You can install and create script for IntelliJ as described in
```

Comment:
Better: You can easily install IntelliJ, create a launch script for it etc. as described in
Or: For installation instructions as well as launch script creation, you can follow the steps described in:

Reply:
Please check again.